scss/bitstyles/tools/_overflow.scss

```diff
@@ -10,9 +10,16 @@
 
 // (1) Fixes bug in safari that causes the scrollbars to appear behind some content within the overflow element.
```

Comment:
Has this bug maybe been fixed, so that we can remove this?

Reply:
Yes it has! 💃